controllers/discordactions.js

```diff
@@ -121,7 +120,41 @@ const addGroupRoleToMember = async (req, res) => {
     return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
   }
 };
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeNicknameOfUsers = async (req, res) => {
+  try {
+    const userData = req.userData;
 
+    const { discordId, username } = userData;
+
+    const discordData = {
+      userName: username,
+      discordId,
+    };
+
+    const authToken = await jwt.sign({}, config.get("rdsServerlessBot.rdsServerLessPrivateKey"), {
+      algorithm: "RS256",
+      expiresIn: config.get("rdsServerlessBot.ttl"),
+    });
+
+    await fetch(`${DISCORD_BASE_URL}/guild/member`, {
+      method: "PATCH",
+      body: JSON.stringify(discordData),
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+    }).then((response) => response.json());
```

Comment:
Any reason why we aren't storing any record of the name-change in our database?

Reply:
it directly changes in discord itself. Rest I received no guideline related to it